WebKitBrowser/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.1.6] - 2023-02-13
```

Comment:
Please increment the patch version, in WebKitBrowser.cpp and here.

Reply:
done